runtime/src/main/java/com/datastax/oss/quarkus/runtime/graal/NativeSubstitutions.java

```diff
@@ -15,20 +15,40 @@
  */
 package com.datastax.oss.quarkus.runtime.graal;
 
+import com.datastax.oss.driver.internal.core.os.CpuInfo;
 import com.datastax.oss.driver.internal.core.os.Native;
+import com.oracle.svm.core.annotate.KeepOriginal;
 import com.oracle.svm.core.annotate.Substitute;
 import com.oracle.svm.core.annotate.TargetClass;
 
 @TargetClass(Native.class)
+@Substitute
 final class NativeSubstitutions {
 
-  /**
-   * This method returns always false because jnr is not available.
-   *
-   * @return false denoting that {@link Native#getProcessId()} cannot be called.
-   */
+  @Substitute private static final CpuInfo.Cpu CPU = CpuInfo.determineCpu();
+
   @Substitute
   public static boolean isGetProcessIdAvailable() {
     return false;
   }
+
+  @Substitute
+  public static int getProcessId() {
+    throw new RuntimeException("Native operations are not supported on this platform");
+  }
+
+  @Substitute
+  public static boolean isCurrentTimeMicrosAvailable() {
+    return false;
+  }
+
+  @Substitute
+  public static long currentTimeMicros() {
+    throw new RuntimeException("Native operations are not supported on this platform");
+  }
+
+  @KeepOriginal
```

Comment:
What does this annotation mean? Why do we need to substitute this method? I think that it does not use any native method.

Reply:
We needed to use a "Substitute" annotation on the class in order to completely replace the old implementation.  The problem here was in the static initialization of Native which means we need to prevent that code from running... and the only way I could see to do that was to replace the class entirely.  If we do that we need to provide an impl of getCpu() in this class, the one that replaces the original Native class.  "KeepOriginal" means "use the impl on the original class"... which is exactly what we want in this case.